src/speciesList.js

```diff
@@ -0,0 +1,258 @@
+// speciesList.js
```

Comment:
Id say all the comments in this file are not really that helpful, they just add noise to the file. Comments should really be used to explain _why_ something is being done the way it is and not _how_ (ideally, the code itself should be clear enough to not need any further explanation). In this case it certainly is